cogs/GlobalConfig.py

```diff
@@ -0,0 +1,73 @@
+import sys, json
+
+class GlobalConfig():
+    def __init__(self):
```

Comment:
I dont think we should be parsing the files in the default constructor, I think it should just setup the class variables and then have a overloaded constructor doing something like
```
def __init__(self, configPath, roleDataPath, lockedDataPath)
    self()
    # parse files here
```

Reply:
Just keep it as storage? Aight

Reply:
I will refactor this slightly in my arg parse PR.